rotkehlchen/tests/unit/accounting/test_basic.py

```diff
@@ -130,11 +130,11 @@ def test_buy_event_creation(accountant):
     assert len(buys) == 2
     assert buys[0].amount == FVal(5)
     assert buys[0].timestamp == 1476979735
-    assert buys[0].rate.is_close('578.6438412')  # (578.505 * 5 + 0.0012 * 578.505)/5
+    assert buys[0].rate.is_close('578.505')  # pure price, fee is counted as a separate event
```

Comment:
As I said this should not be like that. Should be configurable and probably set as default for all to be part of the price.